internal/parser/process.go

```diff
@@ -60,6 +61,18 @@ func ProcessDirectory(directory string) []data.EffxYaml {
 	return yamls
 }
 
+func DetectServicesFromEffxYamls(files []data.EffxYaml, apiKeyString, sourceName string) error {
```

Comment:
do we have a pattern for logging updates so the user for the command line command can see what the cli is doing?

Reply:
Sure, @weeyum added logging